cmd/utils/selector.go

```diff
@@ -42,7 +40,12 @@ const (
 // ErrInvalidOption is raised when the selector has an invalid option
 var ErrInvalidOption = errors.New("invalid option")
 
-// OktetoSelector represents the selector
+// OktetoSelectorInterface represents an interface for a selector
```

Comment:
Maybe this whole file can be moved to its own package named `selector`. it would be great to start emptying the `utils` package.

If you want to do it in this same PR, please, in another commit so we can check the difference between commits

Reply:
i had that in mind, but as it was getting too big of refactor i decided to limit this refactor to make testable the selector logic. This could be a tech-debt issue which i will open and link

Reply:
new issue  https://github.com/okteto/okteto/issues/3043